lib/puppet/functions/extlib/dump_args.rb

```diff
@@ -0,0 +1,20 @@
+require 'json'
```

Comment:
I'm leaning to deprecating this function favor of `echo(to_pretty_json($x))`

Reply:
yeah, probably.  I pretty much picked this function at random when figuring out what the general deprecation pattern might look like.

Reply:
This PR is superseded by https://github.com/voxpupuli/puppet-extlib/pull/120